docs/guides/permissions.md

```diff
@@ -4,4 +4,66 @@ title: Permissions
 
 # Permissions
 
-Documentation coming soon
+Permissions can be managed using `Permission` classes. A `Permission` class extends `BasePermission` and has a `has_permission` method. It can be added to a field using the `permission_classes` keyword argument. A simple example looks like this:
+
+```python
+import strawberry
+from strawberry.permission import BasePermission
+from strawberry.types import Info
+
+class IsAuthenticated(BasePermission):
+    message = "User is not authenticated"
+
+    def has_permission(self, source: Any, info: Info, **kwargs) -> bool:
+        return False
+
+@strawberry.type
+class Query:
+    user: str = strawberry.field(permission_classes=[IsAuthenticated])
+```
+
+If the `has_permission` method fails then an error will be raised using the `message` class attribute. See [Dealing with Errors](/docs/guides/errors) for more information on how errors are handled.
+
+```json
+{
+  "data": null,
+  "errors": [
+    {
+      "message": "User is not authenticated"
+    }
+  ]
+}
+```
+
+
+## Accessing user information
+
+Accessing the current user information to implement your permission checks depends on the web framework you are using. Most frameworks will have a Request object where you can either access the current user directly or access headers/cookies/query parameters to authenticate the user. All the Strawberry integrations will provide this Request object in the `info.context` object that is accessible in every resolver and in the `has_permission` function. You can find more details about a specific framework integration under the "Integrations" heading in the navigation. 
```

Comment:
```suggestion
Accessing the current user information to implement your permission checks depends on the web framework you are using. Most frameworks will have a `Request` object where you can either access the current user directly or access headers/cookies/query parameters to authenticate the user. All the Strawberry integrations provide this Request object in the `info.context` object that is accessible in every resolver and in the `has_permission` function. You can find more details about a specific framework integration under the "Integrations" heading in the navigation. 
```

Reply:
> You can find more details about a specific framework integration under the "Integrations" heading in the navigation

Can we make this a link?

Reply:
@BryceBeagle not at the moment. It's just a heading in the sidebar and not an actual page.